core/include/webview/detail/backends/cocoa_webkit.hh

```diff
@@ -650,11 +595,68 @@ private:
     }
     return temp;
   }
+  void m_window_init() {
```

Comment:
This does what the class constructor and `set_up_window` were doing, just DRYer and much less nesting.